### DISCUSSION TITLE
Support for <animate> tag

### DISCUSSION THREAD

**Discussion by graphics-et-al** [fabricjs/fabric.js#7469] [Ideas]
Hi there!

I've built a little tool that allows users to drag and drop SVG clipart onto a FabricJS canvas, which works pretty well.

Now, I'd like to include some simple animations. I note that the SVG spec allows for this: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/animate

Is there a way for FabricJS to parse this and result in the animated SVG coming across with the animation intact? Or if not, can I make this into a feature request?

Thanks!

**Comment by asturur** [comment]
You can make a feature request but eventually no one will done it.
We are very few people on the project, and most of the time is used on handling issues.
The SVG parser could parse the animate tag, but in the SVG parser roadmap there is first:

- patterns
- masks
- tspan
- groups

On top of that, you should think of what you want to do with those animation once you parse it.
Play them continuosly?
What is the goal / feature set of your tool?

**Reply by asturur** [reply]
I m asking this because even if you get to parse animations, you could be let down by other limitations.

**Reply by graphics-et-al** [reply]
Thanks.
The goal of the tool is similar to Canva (which also runs on FabricJS under the hood :) ), which has animations.
However, we found another (maybe better?) solution using the 'Bodymovin' After effects plugin. See https://codepen.io/shkaper/pen/oEKEgG for an example. This would allow people to use After effects to create importable animations using familiar tools.
